packages/truffle/scripts/patchHighlightJsSolidity.js

```diff
@@ -0,0 +1,32 @@
+const fs = require("fs");
```

Comment:
Can we use `fse`/`fs-extra` here instead? Just for consistency I think we've been using `fse` elsewhere in the codebase.

Reply:
Sure, good catch. Will add.